source/containers/Form/hooks/textReplacement.ts

```diff
@@ -105,15 +106,15 @@ const replaceDates = (descriptor: string[], period?: FormPeriod): string => {
 
   if (descriptor[1] === "currentYear") {
     const year = new Date(
-      activeDate.getFullYear(),
-      activeDate.getMonth() + 1,
+      currentDate.getFullYear(),
+      currentDate.getMonth() + 1,
       1
     ).getFullYear();
     return `${year}`;
```

Comment:
This return statement is a little bit weird, how about:

```js
return year;

instead of:
return `${year}`
```

Reply:
As `.getFullYear()` returns a number, I've opted to call `.toString()` on it.

Reply:
Yeah true. Good catch!